src/components/reader/ReaderSettingsOptions.tsx

```diff
@@ -87,7 +84,22 @@ export function ReaderSettingsOptions({
                         onChange={(e) => setSettingValue('offsetFirstPage', e.target.checked)}
                     />
                 </ListItem>
-            ) : null}
+            )}
+            {(fitPageToWindowEligible && !fitPageToWindow) && (
+              <NumberSetting
+                  settingTitle={t('reader.settings.label.reader_width')}
+                  dialogTitle={t('reader.settings.label.reader_width')}
+                  settingValue={`${readerWidth}%`}
+                  value={readerWidth}
+                  minValue={10}
+                  maxValue={100}
+                  defaultValue={100}
+                  valueUnit="%"
+                  showSlider
+                  handleUpdate={(width: number) => setSettingValue('readerWidth', width)}
+                  sx={{ display: 'flex', alignContent: 'center', justifyContent: 'space-between' }}
```

Comment:
`alignContent` is incrorect, my bad, it has to be `alignItems`

Reply:
Updated